sage/pipeline/pipeline.py

```diff
@@ -1,57 +1,101 @@
+        """
+        self.augmenters = []
         self.config = config
-        self._add_all_augmentors()
+        self._add_all_augmenters()
         if shuffle:
-            self._shuffle_augmentors()
+            self._shuffle_augmenters()
 
-    def _add_all_augmentors(self):
-        self.add_char_augmentor()
-        self.add_word_augmentor()
-        self.add_sbsc_augmentor()
+    def _add_all_augmenters(self):
+        """
+        Adds all available augmenters (character, word, and SBS corruptor) to the pipeline.
+        """
+        self.add_char_augmenter()
+        self.add_word_augmenter()
+        self.add_sbsc_augmenter()
 
-    def _shuffle_augmentors(self):
-        random.shuffle(self.augmentors)
+    def _shuffle_augmenters(self):
+        """
+        Randomly shuffles the order of the augmenters in the pipeline.
+        """
+        random.shuffle(self.augmenters)
 
-    def add_char_augmentor(self):
+    def add_char_augmenter(self):
+        """
+        Adds a character augmenter to the pipeline using the configuration settings.
+        """
         char_config = CharAugConfig(
             min_aug=self.config.char_min_aug,
             max_aug=self.config.char_max_aug,
             unit_prob=self.config.char_unit_prob
         )
-        self.augmentors.append(CharAugmentor(char_config))
+        self.augmenters.append(CharAugmenter(char_config))
 
-    def add_word_augmentor(self):
+    def add_word_augmenter(self):
+        """
+        Adds a word augmenter to the pipeline using the configuration settings.
+        """
         word_config = WordAugConfig(
             min_aug=self.config.word_min_aug,
             max_aug=self.config.word_max_aug,
             unit_prob=self.config.word_unit_prob
         )
-        self.augmentors.append(WordAugmentor(word_config))
+        self.augmenters.append(WordAugmenter(word_config))
 
-    def add_sbsc_augmentor(self):
+    def add_sbsc_augmenter(self):
+        """
+        Adds an SBS corruptor to the pipeline using the configuration settings.
+        """
         sbsc_config = SBSCConfig(
             lang=self.config.sbsc_lang,
             reference_dataset_name_or_path=self.config.sbsc_reference_dataset_name_or_path,
             reference_dataset_split=self.config.sbsc_reference_dataset_split
         )
-        self.augmentors.append(SBSCorruptor(sbsc_config))
+        self.augmenters.append(SBSCorruptor(sbsc_config))
 
-    def remove_augmentor(self, augmentor_type):
-        self.augmentors = [augmentor for augmentor in self.augmentors if not isinstance(augmentor, augmentor_type)]
+    def remove_augmenter(self, augmenter_type):
+        """
+        Removes all instances of the specified augmenter type from the pipeline.
+
+        Args:
+            augmenter_type (type): The type of augmenter to remove.
+        """
+        self.augmenters = [augmenter for augmenter in self.augmenters if not isinstance(augmenter, augmenter_type)]
```

Comment:
When I work with the AugmentationPipeline class and its methods like add_char_augmenter, I don't know about the existence of augmenter types, but I have to get them somewhere